demo-project/README.md

```diff
@@ -4,7 +4,7 @@ This project is designed to be a realistic example of what Kedro looks like when
 
 ## Setup
 
-1. Run `pip install kedro==0.18.4`
-2. Run `kedro install --build-reqs`
+1. Run `pip install kedro~=0.18.0`
+2. Run `pip install -r src/demo-project/requirements.in`
```

Comment:
[Typo in **demo_project**] Can you please change the command to `pip install -r src/demo_project/requirements.in`